README.terraform.md

```diff
@@ -5,9 +5,31 @@ Terraform for running GitLab CI/CD jobs on cloud.gov or another CloudFoundry bas
 * [Deploying](#deploying)
 * [Troubleshooting](#troubleshooting)
 * [Design Decisions](#design-decisions)
+* [Testing](#testing)
 
 ## Deploying
 
+Include this repository as a module in your terraform setup to deploy a gitlab-runner-cloudgov system.
+
+```terraform
+module "gitlab-runner" {
+    source = "github.com/gsa-tts/gitlab-runner-cloudgov?ref=main" # TODO: releases via tags
+
+    cf_space_prefix  = "SPACEPREFIX"
+    ci_server_token  = "TOKEN"
+    docker_hub_user  = "docker-username"
+    docker_hub_token = "docker-personal-access-token"
```

Comment:
[question (nb)] These aren't strictly necessary yet I don't think, should they be marked that way?